docs/installation/linux_and_mac.rst

```diff
@@ -0,0 +1,149 @@
+*********************************
+Installation on Linux and Mac OSX
+*********************************
+
+**Last Updated:** April 2017
+
+Use these instructions to install a development environment on Mac OSX or Linux systems.
+
+.. tip::
+
+    To install and use Tethys Platform, you will need to be familiar with using the command line/terminal. For a quick introduction to the command line, see the :doc:`../supplementary/terminal_quick_guide` article.
+
+1. Download and Run the Installation Script
+-------------------------------------------
+
+For systems with `wget` (e.g. Debian derivatives like Ubuntu):
+
+.. parsed-literal::
+
+      $ wget \https://raw.githubusercontent.com/tethysplatform/tethys/|branch|/scripts/install_tethys.sh
+      $ . install_tethys.sh
+
+For Systems with `curl` (e.g. Mac OSX, and RedHat derivatives like CentOS or Fedora):
+
+.. parsed-literal::
+
+      $ curl \https://raw.githubusercontent.com/tethysplatform/tethys/|branch|/scripts/install_tethys.sh -o ./install_tethys.sh
+      $ . install_tethys.sh
+
+.. note::
+
+    You can customize your tethys installation by passing command line options to the installation script. The available options can be listed by running::
+
+         $ . install_tethys.sh --help
+
+    Each option is also descriped here:
+
+        * `-t, --tethys-home <PATH>`:
+                path for tethys home directory. Default is /usr/lib/tethys on Linux and /Library/Applicaiton/tethys on Mac OS.
+        * `-a, --allowed-host <HOST>`:
+                hostname or IP address on which to server tethys. Default is 127.0.0.1.
+        * `-p, --port <PORT>`:
+                port on which to server tethys. Default is 8000.
+        * `-b, --branch <BRANCH_NAME>`:
+                repository branch to checkout. Default is 'dev'.
+        * `-c, --conda-home <PATH>`:
+                path to conda home directory. Default is ${TETHYS_HOME}/miniconda.
```

Comment:
Does miniconda need to be installed already for this to work? Or does this also install miniconda?